src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,192 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+
+public class Duke {
+
+    private TaskManagement taskManager; // to manage saved data
+    private TaskStorage taskStorage;
+    private Ui ui;
+    public Duke(String filepath) {
+
+        taskStorage = new TaskStorage();
+        ui = new Ui();
+        try {
+            taskManager = new TaskManagement(filepath);
+        } catch (DukeException e) {
+            ui.showLoadingError();
+            taskManager = new TaskManagement();
+        }
+    }
+    public void run() throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        PrintWriter pw = new PrintWriter(new BufferedWriter(new OutputStreamWriter(System.out)));
+        StringBuilder sb = new StringBuilder();
+
+        ui.showWelcome();
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+        String line = "-------------------------------";
+//        System.out.println(line);
+//        System.out.println("Hiii Im\n" + logo);
+//        System.out.println("What can I do for you hmm?");
+//        System.out.println(line);
+        String inp;
+
+        /**
+         * Simply echoes commands entered by the user,
+         * and exits when the user types "bye".
+         */
+        while (true) {
+            inp = br.readLine();
+            String[] input = inp.split(" ");
+            ui.showLine();
+            switch(input[0]) {
+                case "list":
+                    System.out.println("Tasks:");
+                    //System.out.println(taskstorage);
+                    for (int i = 0; i < taskStorage.noTasks(); i++) {
+                        System.out.print(i + 1 + ".");
+                        System.out.println(taskStorage.getTask(i));
+                    }
+                    break;
+
+                case "bye":
+                    System.out.println("Byeee! Hope to see you again! Signing off, duke.Duke.");
+                    break;
+
```

Comment:
Do remember that coding standards state that here should be no indentation for `case` clauses in switch statements

Reply:
Thanks I've rectified it